images/pyspark-notebook/Dockerfile

```diff
@@ -66,9 +66,9 @@ RUN fix-permissions "/etc/ipython/"
 USER ${NB_UID}
 
 # Install pyarrow
-# Temporarily pin pandas to version 1.5.3, see: https://github.com/jupyter/docker-stacks/issues/1924
+# Temporarily pin pandas to version 2.0.3, see: https://github.com/jupyter/docker-stacks/issues/1924
```

Comment:
```suggestion
# Temporarily pin pandas to version 2.0.3, see: https://github.com/apache/spark/pull/42793
```

Reply:
The PR there are for spark 4.0 it will not be backportet to spark 3.5

Reply:
I understand, but the issue in jupyter repo is about pandas 2.0 and it will no longer be relevant after you merge this PR.

Reply:
yes, but what if I change it to 
The pandas version have to be the same as pandas API on spark is build on. To find the right version check out the spark branch you are on and find the pandas version in the file spark/dev/infra /Dockerfile

or 
# NOTE: It's important to ensure compatibility between Pandas versions.
# The pandas version in this Dockerfile should match the version 
# on which the Pandas API for Spark is built.
# 
# To find the right version:
# 1. Check out the Spark branch you are on.
# 2. Find the pandas version in the file spark/dev/infra/Dockerfile.

Reply:
Great idea, thanks.